### ISSUE TITLE
build-sdk.py fails with latest sharp-drm-driver

### ISSUE THREAD

**Issue by nmatsuda** [hack-shack/beepy-directfb2#2]
The script runs fine until the sharp installation script gets called:

```
make: *** No rule to make target 'modules_install'.  Stop.
Traceback (most recent call last):
  File "/home/pi/beepy-directfb2/install-sharp-driver.py", line 140, in <module>
    compile_devicetree_sharp(os.path.join(SRC_DIR,"sharp-drm-driver/"))
  File "/home/pi/beepy-directfb2/install-sharp-driver.py", line 26, in compile_devicetree_sharp
    make("modules_install")
  File "/home/pi/beepy-directfb2/install-sharp-driver.py", line 21, in make
    return subprocess.check_call(['make'] + list(args))
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', 'modules_install']' returned non-zero exit status 2.

```

**Comment by hack-shack** [comment]
**Explanation:** The beepy-directfb2 script, in its current form, does not "pin" its dependencies to specific Github commits. Instead, it uses the latest commit from each dependency. Thus, after ardangelo updated their sharp-drm driver, it broke my script. This could recur in the future as dependencies are updated by their authors - see **Fix** below. Can you kindly attempt the following workaround? In it, you will do a "full" git clone of ardangelo's driver, then you will switch to ("checkout") the last validated version compatible with this beepy-directfb2 script (2023-07-13, 0f82af168d...adf1).
**Workaround:**
`cd beepy-directfb2/src/`
`rm -rf sharp-drm-driver`
`git clone https://github.com/ardangelo/sharp-drm-driver`
`cd sharp-drm-driver`
`git checkout 0f82af168dbd8ec8c7a4e367666b56217863adf1`
`cd ../../`
`python build-sdk.py`
**Fix:** I fixed this bug in the upcoming version of this script (which uses "pinned dependency versions") and will backport the change this week. The spice must flow!

**Comment by nmatsuda** [comment]
hmm throws the same error. But I don't mind waiting till your new version.

**Comment by hack-shack** [comment]
Resolved in latest batch upload (commit 0e4c587a0a6aed88802d09fa49610599bf27a897).

**Comment by nmatsuda** [comment]
confirmed working, thanks!
